src/dotnet/Mgmt.CI.BuildTools/NugetToolsPackage/CI.Tools.Package/Microsoft.Internal.AzNetSdk.CITools.csproj

```diff
@@ -6,7 +6,9 @@
     <Authors>Microsoft</Authors>
     <Description>Build tools for building Azure SDK for NET repository</Description>
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
-    <!-- <PackageLicenseUrl>https://raw.githubusercontent.com/Microsoft/dotnet/master/LICENSE</PackageLicenseUrl> -->
+    <!-- <PackageLicenseUrl>https://raw.githubusercontent.com/Microsoft/dotnet/master/LICENSE</PackageLicenseUrl> 
```

Comment:
Looks like a bad copy/paste.

Reply:
I actually had to delete it, as it kept giving me errors, will remove it completely.